docs/gettinginvolved/infrastructure/packaging.rst

```diff
@@ -186,3 +186,20 @@ To run the ``flocker-control`` container:
         --detach \
         clusterhqci/flocker-control:master
 
+flocker-docker-plugin
+--------------------
+
+The ``flocker-docker-plugin`` Docker image is built using the same ```docker build ...``` command line as for ``flocker-dataset`` but substituting the ```docker-plugin/Dockerfile```.
```

Comment:
❓ Nit. The substitution is actually ``docker-plugin``, you don't give the path to the `Dockerfile`
Also, use double backticks not triple backticks for inline code: 
 * http://clusterhq-staging-docs.s3-website-us-east-1.amazonaws.com/docker-image-flocker-docker-plugin-FLOC-4540/gettinginvolved/infrastructure/packaging.html#flocker-docker-plugin